app/models/decidim/participations/participation.rb

```diff
@@ -23,11 +23,24 @@ class Participation < Participations::ApplicationRecord
 
       geocoded_by :address, http_headers: ->(participation) { { "Referer" => participation.feature.organization.host } }
 
+      # upstream moderation => MOA dashboard
+      scope :unmoderate, ->(current_feature) { current_feature_participations(current_feature).joins(:moderation).merge(Moderation.where(upstream_moderation: "unmoderate")) }
+      scope :moderated, -> (current_feature){ current_feature_participations(current_feature).joins(:moderation).merge(Moderation.where('upstream_moderation = ? OR upstream_moderation = ?', 'authorized', 'refused')) }
+      scope :questions_with_unpublished_answer, -> (current_feature) { current_feature_participations(current_feature).joins(:moderation).merge(Moderation.where.not(['upstream_moderation = ? OR upstream_moderation = ? OR upstream_moderation = ?', 'unmoderate', 'authorized', 'refused'])) }
+
+      # useless for this feature
```

Comment:
if useless for this feature, should we remove it?

Reply:
For the moment we'll keep it here, just to avoid breaking things